pkg/cmd/actions/actions.go

```diff
@@ -34,28 +34,45 @@ func NewCmdActions(f *cmdutil.Factory) *cobra.Command {
 	return cmd
 }
 
-func actionsRun(opts ActionsOptions) {
-	cs := opts.IO.ColorScheme()
-	fmt.Fprint(opts.IO.Out, heredoc.Docf(`
-			Welcome to GitHub Actions on the command line.
+func actionsExplainer(cs *iostreams.ColorScheme, color bool) string {
+	header := "Welcome to GitHub Actions on the command line."
+	runHeader := "Interacting with workflow runs"
+	workflowHeader := "Interacting with workflow files"
+	if color {
+		header = cs.Bold(header)
+		runHeader = cs.Bold(runHeader)
+		workflowHeader = cs.Bold(workflowHeader)
+	}
 
-			This part of gh is in beta and subject to change!
+	return heredoc.Docf(`
+			%s
 
-			To follow along while we get to GA, please see this
-			tracking issue: https://github.com/cli/cli/issues/2889
+			gh integrates with Actions to help you manage runs and workflows.
```

Comment:
```suggestion
			GitHub CLI integrates with Actions to help you manage runs and workflows.
```

Feels a little more grammatically correct

Reply:
ah oops i didn't see this comment -- i'll sneak this into another PR.